src/System.Net.Http/src/uap/System/Net/HttpClientHandler.cs

```diff
@@ -259,18 +259,34 @@ public int MaxConnectionsPerServer
         
         public long MaxRequestContentBufferSize
         {
-            get { return HttpContent.MaxBufferSize; }
+            // This property has been deprecated. In the .NET Framework it was only used when the handler needed to 
```

Comment:
There's no problem obsoleting in the REF assembly: https://github.com/dotnet/corefx/blob/master/src/System.Net.Primitives/ref/System.Net.Primitives.cs#L469

Then what I'm saying is that the comment seems confusing: it's either deprecated and has the Obsolete attribute so that the developers know at compile time not to use it anymore (via a warning which can easily be disabled) or it's just not supported by this platform. Maybe remove the word "deprecated" until we have that discussion?